app/actions/userActions.js

```diff
@@ -24,7 +51,37 @@ export function getUser(id) {
         return userService.getUser(id)
             .then(user => dispatch(getUserSuccess(user)))
             .catch(error => {
-                throw(error);
+                throw (error);
```

Comment:
What's this pattern used for? A rejected non-caught promise would throw anyway, wouldn't it? Also, this will throw in another event loop, so will anything catch this throw? It is just another promise being rejected.